src/TIOperation.php

```diff
@@ -31,12 +30,11 @@ class TIOperation {
     private $date;
     private $operationType;
     
-    function __construct($id, $status, $trades, $commisionCurrency, $commisionValue, $currency, $payment, $price, $quantity, $figi, $instrumentType, $isMarginCall, $date, $operationType) {
+    function __construct($id, $status, $trades, $commission , $currency, $payment, $price, $quantity, $figi, $instrumentType, $isMarginCall, $date, $operationType) {
         $this->id = $id;
         $this->status = $status;
         $this->trades = $trades;
-        $this->commisionCurrency = $commisionCurrency;
-        $this->commisionValue = $commisionValue;
+        $this->commission = $commission;
```

Comment:
You have to remove getCommisionCurrency and getCommisionValue also
and create getComission method

Reply:
yes